exercises/pythagorean-triplet/src/Triplet.hs

```diff
@@ -1,7 +1,10 @@
 module Triplet (isPythagorean, mkTriplet, pythagoreanTriplets) where
 
-isPythagorean = error "You need to implement this function."
+isPythagorean :: (Int, Int, Int) -> Bool
```

Comment:
> Why did we give this choice, anyway?

I didn't see explanation given in https://github.com/exercism/exercism.io/pull/741/files. I guess we can see that `mkTriplet` is invariant to ordering of the three numbers, since it will always order them in a consistent way.

So we still need `mkTriplet`.